src/scss/styles.scss

```diff
@@ -832,3 +833,16 @@ img.logo {
 .cursor-move {
     cursor: move !important;
 }
+
+.col-3 {
+    max-width: 240px;
```

Comment:
Bootstrap uses flex-box and `%` for controlling `col-*` widths in a fluid layout. I wouldn't want to cap those simply because of the bootstrap styles that are being used for the web vault's layout without more specificity. If there are regions/parts of the web vault's layout that need to be capped on width that are currently only styles as `col-3`, for example the **FILTERS** sidebar, that should then get an additional css class appended to the markup itself, to be targeted by the stylesheet specifically:
```less
.left-nav {
    max-width: 240px;
}
```
OR, the use of specificity, nesting and *nth* operators be used for targeting those elements, i.e.
```less
.container {
    /*...*/
    .row {
        .col-3 {
            &:first-child {
                max-width: 240px;
            }
        }
    }
}
```

Reply:
Changes made as requested.